lxc/publish.go

```diff
@@ -213,6 +216,14 @@ func (c *cmdPublish) Run(cmd *cobra.Command, args []string) error {
 		req.Public = c.flagMakePublic
 	}
 
+	if c.flagExpiresAt != "" {
+		expiresAt, err := time.Parse(time.RFC3339, c.flagExpiresAt)
+		if err != nil {
+			return err
```

Comment:
@masnax this is just a minor nit, but as its you're first PR (I think) I'll mention it.

Depending on how "good" the error returned from `time.Parse` is here, from a user experience perspective, it may be warranted to wrap the error using the "github/errors.com/pkg" package which we use a lot in LXD, e.g.

```
return errors.Wrapf(err, "Invalid expiration date")
```

Reply:
Absolutely! I was debating this, but instead opted to handle errors like its immediate neighbours.
`time.Parse` returns an error complaining that the input value doesn't match the RFC3339 template. While it can be inferred that it's talking about an invalid date, it's never explicitly mentioned, so I'll add the change!

Reply:
Yeah there's quite a lot of existing places that could do with additional context in the errors to improve usability.